lab_4_fill_words_by_ngrams/main.py

```diff
@@ -114,6 +185,9 @@ def __init__(self) -> None:
         """
         Initialize an instance of GeneratorTypes.
         """
+        self.greedy = 0
```

Comment:
And for what did you create these fields?

Reply:
fixed? I wanted it to use it for get_conversion_generator_type function.